examples/asa/scripts/2-gold-asc.js

```diff
@@ -38,7 +44,9 @@ async function run(runtimeEnv, deployer) {
 	const lsig = await deployer.getLsig("Gold_C_Lsig");
 	const goldAsset = deployer.asa.get("gold");
 	const goldAssetID = goldAsset.assetIndex;
-	await deployer.optInLsigToASA(goldAssetID, lsig, { totalFee: 1000 });
+	await deployer.optInLsigToASA(goldAssetID, lsig, { totalFee: 1000 }).catch((error) => {
```

Comment:
Hm, it will not fail in the CI if the error is thrown in deployer maybe we should just do assert.notThrow? It looks a bit strange the construction here